pkg/kudoctl/cmd/diagnostics/diagnostics.go

```diff
@@ -0,0 +1,123 @@
+package diagnostics
+
+import (
+	"github.com/spf13/afero"
+
+	"github.com/kudobuilder/kudo/pkg/kudoctl/env"
+	"github.com/kudobuilder/kudo/pkg/version"
+)
+
+const (
+	nsKudoSystem      = "kudo-system"
+	labelKudoOperator = "kudo.dev/operator"
+	appKudoManager    = "kudo-manager"
+)
+
+const (
+	continueOnError = false
+	failOnError     = true
```

Comment:
IMO, it's not necessary to define this here. This is used when calling functions that have a `failOnError` parameter that should be enough to document the intent.

Reply:
On the other hand, reading `continueOnError` and `failOnError` in the code below _is_ nice. Feel free to keep this 😄

Reply:
We would not need the constants if we could call funcs with named params, just like we initialize structs, but in this case I tend to keep the constants. This way the reader knows the meaning without looking at the function definition. It matches your second comment.